generator/sequence_transform_data.py

```diff
@@ -421,72 +412,117 @@ def serialize_sequence_trie(
     """
     table = []
 
+    def build_chain_match(sub_rule, match):
+        return {
+            'SUB_RULE': sub_rule,
+            'DATA': build_match(match)
+        }
+
+    def build_match(match, more_branching = False):
+        global max_backspaces
+
+        action = match['ACTION']
+
+        backspaces = action['BACKSPACES']
+        max_backspaces = max(max_backspaces, backspaces)
+        func = action['FUNC']
+        completion = action['COMPLETION']
+        completion_index = completions_map[completion]
+
+        # 1 bit reserved for marking if trie branching continues
+        code = 0x80 if more_branching else 0
```

Comment:
This bit currently serves no purpose, but it might be needed by RS_new. If it turns out that it isn't needed, we can repurpose it for something else.